CHANGELOG.rst

```diff
@@ -11,6 +11,7 @@ This project adheres to `Semantic Versioning`_ starting with version 1.0.
 
 Added
 -----
+- added agent readiness check to the `/status` resource
```

Comment:
```suggestion
- added agent readiness check to the ``/status`` resource
```

Reply:
Doesn't that have the exact same result?
Maybe I'm missing something.

Reply:
That's just the practise we are following, but I think it's also interpreted slightly differently: https://github.com/ralsina/rst-cheatsheet/blob/master/rst-cheatsheet.rst